### ISSUE TITLE
Ubuntu GNS3 - unable to console 

### ISSUE THREAD

**Issue by SNV100** [GNS3/gns3-server#1507]
Hi
I am running GNS3 on Ubuntu 18.04.1 LTS. Unable to console to Cisco Images. Terminal opens with console port - does not show up the console

- Image is verified with a  GNS3 VM - Working fine with out any issues. 
- UFW - is in inactive state

 

GNS3 Server version 
   INFO run.py:213 GNS3 server version 2.1.12

Dynamips : 
   Cisco Router Simulation Platform (version 0.2.19-amd64/Linux stable)
   Copyright (c) 2005-2011 Christophe Fillot.
   Build date: Jan 19 2019 23:00:00

The console command stops with a message below. 
    Connected to Dynamips VM (ID 2, type c3745) - Console port
     Press ENTER to get the prompt.

**Comment by grossmj** [comment]
You may want to try this. Uninstall your Dynamips 64-bit version and install the 32-bit one:

```
sudo add-apt-repository ppa:gns3/ppa
sudo dpkg --add-architecture i386
sudo apt update
sudo apt-get install gns3-gui
sudo apt install dynamips:i386
```

Please let me know if this solves the issue.

**Comment by SNV100** [comment]
Hi , 

I remember running this on ver 16 - Currently I have a new ubuntu machine. 

Tried the following 
sudo apt remove dynamips
sudo add-apt-repository ppa:gns3/ppa
sudo dpkg --add-architecture i386
sudo apt update
sudo apt-get install gns3-gui
sudo apt install dynamips:i386

Still the same issue 

The comamnd sudo apt remove dynamips removed a package called gns3 as well 
When I tried installing this using sudo apt-get install gns3,  

The following additional packages will be installed:
  dynamips
The following packages will be REMOVED:
  dynamips:i386
The following NEW packages will be installed:
  dynamips gns3

I did not proceed further.

**Comment by grossmj** [comment]
We have released Dynamips 0.2.20, please update your Debian packages and try to install dynamips again.

Please comment back if you still have problems.
